migrations/sqlite/20240529202212_stream.up.sql

```diff
@@ -0,0 +1,8 @@
+-- Add up migration script here
+CREATE TABLE IF NOT EXISTS "ceramic_one_stream" (
+    "cid" BLOB NOT NULL, -- init event cid
+    "sep" TEXT NOT NULL, 
```

Comment:
yeah, I wanted to discuss controller and whether we want to fully normalize it out. it's easy to capture for just the init event here, but we may want to include it for every event as well. I wasn't sure if we wanted many to many or just a single one. having it normalized for every event makes it possible to do "flat recon" things but it would still be a decent amount of work to implement.

Reply:
I think for now we should assume that there is only ever a single controller, and that the controller for a stream never changes.  Both of those are true today, and if either were to ever change it would require work across the entire stack, so I think requiring a database migration if/when that were to happen would be okay.  For now let's keep it simple.